reconfigure/coordinator.js

```diff
@@ -26,19 +27,20 @@ Coordinator.prototype.list = cadence(function (async) {
 Coordinator.prototype.update = cadence(function (async) {
     async(function () {
         this._consensus.listeners(async())
-    }, function (list) {
-        async.forEach(function (urls) {
-            /*
+        this.list(async())
+    }, function (urls, list) {
+        async.forEach(function (url) {
             async(function () {
-                // http POST and service is missing
+                this._ua.update(url, list, async())
```

Comment:
Let's update `UserAgent.update` to return the `response.okay` from `Vizsla`. That way we have a boolean to determine if the procedure succeeded.